addons/vehicles/script_component.hpp

```diff
@@ -0,0 +1,16 @@
+#define COMPONENT Vehicles
```

Comment:
```suggestion
#define COMPONENT vehicles
```

Reply:
Ahh yeah I misread the definition in /Weapons 😅